hack/local-up-cluster.sh

```diff
@@ -147,6 +147,14 @@ function test_docker {
     fi
 }
 
+function test_cfssl {
+    if ! cfssl &>/dev/null || ! cfssljson &>/dev/null; then
```

Comment:
`cfssljson` seems to hang for me (probably because it waits for something in stdin). How about using `which` or something else to verify the binary exists?